apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/BorrowDetailsGraph.tsx

```diff
@@ -125,9 +148,8 @@ export const BorrowDetailsGraph: FC<BorrowDetailsGraphProps> = ({
 
         <Chart
           input={{
-            // TODO: implement once data is available
-            data: [],
-            label: '',
+            data,
+            label: t(pageTranslations.chart.label1),
```

Comment:
`label1` what is it? we're tipically setting the text or shorter version of it, if it's `"Supply Apr"` key can be `supplyApr` or `supplyAprLabel`

Reply:
certanly ; i brought label1 from chart component from 2_molecules but agree with you to rename it